azure/stack/setup/README.md

```diff
@@ -0,0 +1,9 @@
+### Important
+
+Before executing any Terraform commands, it's essential that you obtain your user object ID. This ID is required to ensure proper access and permissions within Terraform. Please follow these steps:
+
+Open your command line interface.
+
+1. Obtain your object ID
+`az ad signed-in-user show --query id --output tsv`
```

Comment:
```suggestion
1. Obtain your object ID

```shell
az ad signed-in-user show --query id --output tsv
```

```

easier to read, and on Github, you get a "copy" button :)